frameworks/Python/uvicorn/uvicorn_conf.py

```diff
@@ -0,0 +1,13 @@
+import multiprocessing
+import os
+
+_is_travis = os.environ.get('TRAVIS') == 'true'
+
+workers = multiprocessing.cpu_count()
+if _is_travis:
+    workers = 2
+
+bind = "0.0.0.0:8080"
+keepalive = 120
+errorlog = '-'
+pidfile = 'uvicorn.pid'
```

Comment:
hey @tomchristie Thanks for the submission. One of the issues we're having with a few frameworks is where they are writing certain files. Where is `unicorn.pid` being stored? If it's within the framework directory, can we point it to `/tmp` or `$IROOT`?